tests/test.py

```diff
@@ -1548,8 +1548,7 @@ def test_only_alpha_client_id(self):
         # call with directory separator in key name
         self.assertEqual(create_client_keypair(POISON_KEY_PATH), 1)
 
-
-class SSLPostgresqlConnectionTest(AcraCatchLogsMixin, HexFormatTest):
+class SSLPostgresqlMixin(AcraCatchLogsMixin):
     ACRA2_PORT = BaseTestCase.ACRA_PORT+1000
```

Comment:
Why did you decide to plus 1000? Why not 2000 or 10000?

Reply:
chosen manually. we ran tests on ~5-10 go versions and for each we use ACRA_PORT+1. Thought that 100 may have intersection with any PROXY_PORT

Reply:
There is no probability that port will be greater than max value - 65535?

Reply:
No, because it may be overrided with TEST_ACRA_PORT env variable that in tests use 10003 port as default